crates/subspace-farmer-components/Cargo.toml

```diff
@@ -32,7 +32,7 @@ subspace-solving = { version = "0.1.0", path = "../subspace-solving" }
 subspace-core-primitives = { version = "0.1.0", path = "../subspace-core-primitives" }
 subspace-verification = { version = "0.1.0", path = "../subspace-verification" }
 thiserror = "1.0.38"
-tokio = { version = "1.25.0", features = ["macros", "parking_lot", "rt-multi-thread", "signal"] }
+tokio = { version = "1.25.0", features = ["macros", "parking_lot", "rt-multi-thread", "signal", "sync"] }
```

Comment:
Totally understandable. The feature was missing for a long time, but we didn't notice it when running all tests together, in which case it was added anyway due to feature unification. But when I tried to run just the tests of `sp-lightclient` alone, it failed to compile because this feature was used. but not specified.